src/libponyc/type/alias.c

```diff
@@ -186,8 +187,12 @@ static ast_t* consume_single(ast_t* type, token_id ccap)
   ast_t* cap = cap_fetch(type);
   ast_t* eph = ast_sibling(cap);
   token_id tcap = ast_id(cap);
+  token_id teph = ast_id(eph);
+  //cap_aliasing(&tcap, &teph);
```

Comment:
Please ensure this line eventually gets either uncommented or removed.

Reply:
@jemc given the tests are passing, do you know f it is reasonable to remove this?

Reply:
No, I don't know - would prefer to get Jason's input.

Reply:
I ran into bugs having this, and I'm not entirely sure whether it's necessary to have as it depends on how this method gets called.

If you do include the line one of the stdlib packages fails because it tries to subtype `#alias^` as `#any`, iirc

So we can remove it so long as all the normal tests are passing.

Reply:
Ok. removing.